packages/sprintf/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.1.0 (2019-07-25)
```

Comment:
Both here and in #2, can you leave the date as unset? I'm not sure I can commit to having this published today.

```suggestion
## 1.1.0 (TBD)
```

Reply:
Can do!